docs/containers.config.5.1

```diff
@@ -0,0 +1,131 @@
+.TH "containers.conf" "5" " Container runtime tools configuration file" 
```

Comment:
This file is a build artifact and should not be in the repo.  We should only have containers.config.5.md